FirebaseAppCheck/Sources/Core/GACAppCheckSettings.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Google LLC
```

Comment:
Should GACAppCheckSettings.h be in a new folder hierarchy to better correspond with its future migration - like FirebaseAppCheck/Sources/GAC/Sources/Core/ ?